replay/src/main.rs

```diff
@@ -441,6 +457,86 @@ fn main() {
 
             save_entry_point_execution(&path, block_executions).unwrap();
         }
+        ReplayExecute::Call {
+            call_path,
+            tx,
+            block_number,
+            chain,
+        } => {
+            let mut state = build_cached_state(&chain, block_number - 1);
+            let reader = build_reader(&chain, block_number);
+
+            let call_file = File::open(call_path).unwrap();
+            let call: ExecutableCallEntryPoint = serde_json::from_reader(call_file).unwrap();
+
+            // We fetch the compile class from the next block, instead of the current block.
+            // This ensures that it always exists.
+            let compiled_class = reader.get_compiled_class(call.class_hash).unwrap();
+
+            // This mocked context was built from trial and error. It only sets
+            // the required field to execute a sample transaction, but probably
+            // won't work in every escenario. For example, if a transaction
+            // depends on a particular value of the context, it would probably
+            // fail.
+            //
+            // The actual solution is to save the exact execution context from
+            // the original complete execution, and use it here, restoring every
+            // single field. For the sake of simplicity I didn't do it, but it
```

Comment:
Maybe something like this?

```suggestion
            // single field. This was not chosen as the current implementation for the sake of simplicity, but it
```